Update storage-use-azcopy-linux.md

mabicca opened the pull request:
Fixing typo about BlobType parameter. There is no /BlobType in azcopy for Linux, there is --blob-type:

According to the help from the command itself:
azcopy --help | grep -i page
--blob-type <page | block | append>

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Blackmist commented:
Looks good, merge.

mabicca committed:
Update storage-use-azcopy-linux.md

Fixing typo about BlobType parameter. There is no /BlobType in azcopy for Linux, there is --blob-type:

According to the help from the command itself:
azcopy --help | grep -i page
--blob-type <page | block | append>
